src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue8845.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using UITest.Appium;
+using UITest.Core;
+
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue8845 : _IssuesUITest
+	{
+		public Issue8845(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "Picker on windows shows \"Microsoft.Maui.Controls.Picker\" if ItemsSource has an empty string"; 
+		
+		[Test]
+		[Category(UITestCategories.Picker)]
+		[FailsOnMac]
+		public void PickerShouldDisplayValueFromItemDisplayBinding()
+		{
+			App.WaitForElement("UpdateButton");			
+			App.Tap("UpdateButton");
+
+			var picker1Text = App.WaitForElement("MauiPicker1").GetText();
+			var picker2Text = App.WaitForElement("MauiPicker2").GetText();
```

Comment:
Can you verify what this text is vs just confirming that it's not null? 

Also, instead of using ClassicAssert can you use the newer syntax?

```C#
Assert.That(picker2Text, Is.EqualTo(....));
```

Reply:
I have updated the test cases as mentioned above, could you please verify once and let me know.